src/applications/mhv/landing-page/tests/e2e/restricted-by-log-in-provider/auth.provider.cypress.spec.js

```diff
@@ -1,32 +1,38 @@
 import { CSP_IDS } from '@department-of-veterans-affairs/platform-user/authentication/constants';
-import manifest from '../../manifest.json';
+import { appName } from '../../../manifest.json';
+import vamcEhr from '../../fixtures/vamc-ehr.json';
 
 import ApiInitializer from '../utilities/ApiInitializer';
 import LandingPage from '../pages/LandingPage';
 
-describe(manifest.appName, () => {
+describe(appName, () => {
   beforeEach(() => {
+    cy.intercept('GET', '/data/cms/vamc-ehr.json', vamcEhr).as('vamcEhr');
```

Comment:
MINOR REFACTOR: I would consider using the ApiInitializer class to keep your intercepts in one place that is reusable 

https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/mhv/landing-page/tests/utilities/ApiInitializer.js

Reply:
Good call. I'll be sure to use and update the ApiInitializer in future PRs.